terraform/monitoring/panels/app/publishing_workers_count.libsonnet

```diff
@@ -0,0 +1,19 @@
+local grafana   = import '../../grafonnet-lib/grafana.libsonnet';
+local defaults  = import '../../grafonnet-lib/defaults.libsonnet';
+
+local panels    = grafana.panels;
+local targets   = grafana.targets;
+
+{
+  new(ds, vars)::
+    panels.timeseries(
+      title       = 'Publishing worker tasks count',
+      datasource  = ds.prometheus,
+    )
+    .configure(defaults.configuration.timeseries)
+    .addTarget(targets.prometheus(
+      datasource  = ds.prometheus,
+      expr        = 'sum(rate(spawned_publishing_workers{}[$__rate_interval]))',
```

Comment:
```suggestion
      expr        = 'sum(rate(spawned_publishing_workers_total{}[$__rate_interval]))',
```

Reply:
Yep, I've missed it. Thanks for catching this!

Reply:
Fixed.